assets/i18n/en_US.json

```diff
@@ -111,6 +111,8 @@
 
         "downloadToast": "Download function is not available yet",
 
+        "requireSuggestedAppVersionDialogText": "The version of the app you have selected does not match the suggested version. Please select the app that matches the recommended version.\n\nSelected version: v{selected}\nSuggested version: v{suggested}\n\n.You can disable \"Require suggested app version\" in the settings to bypass this.",
```

Comment:
I see some consistencies here. Please replace `the recommended version` with `the suggested version`.

and for the last sentence, I think `You can turn "Require suggested app version" off in settings if you want to proceed anyways.` is a better fit.

Reply:
```suggestion
        "requireSuggestedAppVersionDialogText": "The version of the app you have selected does not match the suggested version. Please select the app that matches the suggested version.\n\nSelected version: v{selected}\nSuggested version: v{suggested}\n\n.To proceed anyways, disable "Require suggested app version" in the settings.",
```